content/docs/dvclive/api-reference/live/make_summary.md

```diff
@@ -0,0 +1,37 @@
+# Live.make_summary()
+
+Serializes a summary of the logged metrics (`Live.summary`) to
+`Live.metrics_file`.
```

Comment:
Not blocking:

```suggestion
Saves a summary of the latest metrics values (`Live.summary`) to
`dvclive/metrics.json`.
```

I think it's fine to either use `dvclive/metrics.json` or keep `Live.metrics_file` and link back to where it's explained.